tests/x11regressions/gnomecase/application_starts_on_login.pm

```diff
@@ -103,7 +115,10 @@ sub run {
     send_key "alt-f4";
 
     handle_logout;
+    # The keyboard focus was losing in gdm of SLE15 bgo#657996
+    mouse_set(520, 350) if sle_version_at_least('15');
```

Comment:
I'm not sure if it's a good idea, hard-coded coordinates in 2 places... Can we replace if with assert_and_click or some key presses?

Reply:
Also you have the same multiple times. Please avoid the duplication

Reply:
Updated

- Verification run: http://10.67.17.30/tests/63